pkg/cmd/kafka/topic/update/update.go

```diff
@@ -202,6 +203,14 @@ func runCmd(opts *Options) error {
 				return err
 			}
 		}
+
+		if opts.partitionsStr != "" {
+			partitionCount, err = topicutil.ConvertPartitionsToInt(opts.partitionsStr)
+			if err != nil {
+				return err
```

Comment:
This could be some generic error from system that is not i18n
Best to return explicit error.

Reply:
The validators return i18n error messages.